src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -31,7 +32,8 @@ export class ProjectsSummaryController implements angular.IController {
   private AuthService: any;
   private DataService: any;
   private Constants: any;
-  private RecentlyViewed: any;
+  private RecentlyViewedItems: any;
+  private RecentlyViewedProjects: any;
```

Comment:
Nit: I'd rather make it match the service name exactly, so `RecentlyViewedProjectsService` here and below.